app/controllers/catalog_controller.rb

```diff
@@ -216,6 +216,22 @@ def active_sort_fields
     # config.add_results_document_tool(:bookmark, partial: 'bookmark_control', if: :render_bookmarks_control?)
 
     config.add_results_collection_tool(:sort_widget)
+
+
+    # See also:
+    # app/models/search_builder/all_search_result_ids_builder.rb
+    # app/controllers/all_search_result_ids_controller.rb
+    #
+    # Don't show this tool on e.g. the collection results page or the featured topics page.
+    config.add_results_collection_tool(
+      :all_search_results_add_to_cart,
+      opts = {
+        if: proc { |controller| controller.can?(:update, Kithe::Model) && controller.class.name == 'CatalogController' }
+      }
+    )
```

Comment:
Oh my, this is a part of Blacklight I have literally _never_ seen before! Good job figuring it out, that was some real code spelunking! 

I hope it doens't end up fragile towards future BL upgrades... I suppose we will just have to deal with it if it does.